google/cloud/bigtable_v2/__init__.py

```diff
@@ -122,6 +122,8 @@
     "RowSet",
     "SampleRowKeysRequest",
     "SampleRowKeysResponse",
+    "ExecuteQueryRequest",
+    "ExecuteQueryResponse",
```

Comment:
THis file is autogenerated and doesn't need to be touched (These are already included in lines 87/88)

Reply:
removed.